### ISSUE TITLE
spec: clarification for event handlers

### ISSUE THREAD

**Issue by mpasternacki** [appc/spec#58]
1. If multiple event handlers of the same name are defined, should they be executed in parallel, or one after another?
2. Are event handlers executed as root, or as parent application's user/group? A good use case for handlers would be privileged initialization (e.g. creating and `chown`-ing workspace for process, which requires root) to let the main process run as a non-privileged user.
3. Do event handlers run with parent app's environment, or with a clean one?

It may be useful to have event handlers more parameterized: it should be possible to specify at least user, group, and environment for the handler. If you are not opposed to that idea, I will prepare a pull request.

**Comment by jonboulle** [comment]
Closing this in favour of the slightly more specific #188
